src/pages/mentorship/index.tsx

```diff
@@ -1,13 +1,30 @@
 // path: /mentorship
+import { GetServerSideProps } from 'next';
 
-import { Typography } from '@mui/material';
+import { MentorshipPageProps } from '@utils/types';
+import { fetchData } from 'lib/api';
 
-const MentorshipPage = () => {
-  return (
-    <div>
-      <Typography variant="h4">Welcome to the Mentorship Page</Typography>
-    </div>
-  );
+const MentorshipPage = ({ mentorship }: MentorshipPageProps) => {
+  return <div>{JSON.stringify(mentorship)}</div>;
+};
+
+export const getServerSideProps: GetServerSideProps = async () => {
+  try {
+    const combinedResponse = await fetchData('mentorship/overview');
+
+    return {
+      props: {
+        mentorship: combinedResponse.data,
```

Comment:
Not sure about the design, but is there a footer here - the footer can be added the same way as in the index page

Reply:
In the design doesn't appear any footer, how do I know that I need to include footer? Yes, I can add the footer.

Reply:
No need to add the footer if it's not in the design then